vpc/service/assign_addresses_v3.go

```diff
@@ -298,6 +298,10 @@ WHERE assignment_id = $1
 				tracehelpers.SetStatus(err, span)
 				return nil, err
 			}
+		} else {
+			err = status.Errorf(codes.FailedPrecondition, "Assignment %s is not completed", assignmentID)
```

Comment:
This error indicates that we tried to assign an ip, but there was already some unfinished (incomplete) assignment in progress or something?

Reply:
Right, and that the assignment is either in the process of being completed, or has failed to complete.